api/tests/certification/session-management/unit/domain/usecases/supervise-session_test.js

```diff
@@ -10,7 +10,7 @@ import { catchErr, domainBuilder, expect, sinon } from '../../../../../test-help
 
 describe('Unit | UseCase | supervise-session', function () {
```

Comment:
Va probablement aussi falloir renommer le usecase

```suggestion
describe('Unit | UseCase | invigilate-session', function () {
```